### ISSUE TITLE
[REQUEST] Implement mGBA 0.8.1 core (maybe as hotfix?)

### ISSUE THREAD

**Issue by NotFx** [TASVideos/BizHawk#1877]
The mGBA 0.8 core has a multitude of very noticeable bugs, making accurate TASing on it practically impossible. 0.8.1 fixes many of these issues. I'm hoping it can be implemented in an official release in the near future?

**Comment by vadosnaprimer** [comment]
You mean "hey just release already"?

**Comment by NotFx** [comment]
> 
> 
> You mean "hey just release already"?

From that response I'm guessing there's things being worked on that are preventing that (I wouldn't know, I don't code), but asking can't hurt. Thanks anyway.

**Comment by endrift** [comment]
How many of those bugs weren't fixed in 0.8.1? How many of those bugs weren't reported so I have no way of knowing about them, much less fixing them?

**Comment by NotFx** [comment]
> 
> 
> How many of those bugs weren't fixed in 0.8.1? How many of those bugs weren't reported so I have no way of knowing about them, much less fixing them?

Didn't mean to make it sound like your work isn't appreciated; mGBA is still the best program by far.
If I run into anything apparent, I'll be sure to write it to the mGBA github. So far on 0.8.1, the games I usually work on seem to run well.

**Comment by endrift** [comment]
For what it's worth, interim builds are currently tracking master (which is post-0.8.1); we just need to stabilize some of the changes first since @vadosnaprimer and I are working on bringing the core in line with the VBA-Next core in terms of features (see #1777)

**Comment by Spikestuff** [comment]
Let me summarize the response you got @NotFx.
@vadosnaprimer always updates the mGBA core when a release of Hawk is about to come out.
If it's not out, you look at devbuilds sometimes there's things that @endrift notices that vado acts on and they get updated there.

TL;DR: At least the mGBA core gets updated compared to others and your requested is already [committed](https://github.com/TASVideos/BizHawk/commit/b07f8db8c9cd34dd4fd04b6bb305eaaccfcb09b2).

**Comment by RetroEdit** [comment]
Funny enough, I just found a problem in the BizHawk 2.4's version of mGBA core. If you play Final Fantasy I (in Final Fantasy I & II Dawn of Souls) and try to enter the menu by pressing 'Start', the game crashes. This issue is fixed in the BizHawk's development build.

**Comment by YoshiRulz** [comment]
Pulled submodule and updated core dll: da584d1a6
